Kickstarter CrazyBaby landing

BudnikOleksii opened the pull request:
- [DEMO LINK](https://BudnikOleksii.github.io/Kickstarter/)

ivan-kadykalo reviewed (changes requested):
Переглянув твій лендос, є декілька коментів, але загалом виглядає все класно.
https://www.loom.com/share/19286520921742a1ada95be8ec62d864

Цей блок таки зʼїхав на широкому екрані
![image](https://user-images.githubusercontent.com/57747481/174038311-5e0643f7-3849-4c13-80c3-f687cc11ca69.png)

Ihor-Karpyn reviewed (changes requested):
- #### U can create better hover effects, try to find something interesting on google
----
![image](https://user-images.githubusercontent.com/63306833/174035677-a2d7c509-0bc1-40ce-b332-3ad00aa47b21.png)
----

- #### Something wrong with the image background, u downloaded not right file type, this one is without transparent background

![image](https://user-images.githubusercontent.com/63306833/174036176-968ac165-9847-4fe4-9a54-dc4c3e5df37c.png)

- #### Something wrong with the container

![image](https://user-images.githubusercontent.com/63306833/174036805-74dd50a4-5fca-40e1-8277-904274d39e0e.png)

- #### Be more attentive to this menu, the cross is white, the text is dark, and what arrow doing here? And the gap between nav items looks streng

![image](https://user-images.githubusercontent.com/63306833/174037546-97d546cb-49e0-4a0b-9147-e27ad38ddc1c.png)


- #### U have a lot of time, try to add a slider on mobile. You can create it by [js](https://kenwheeler.github.io/slick/), [CSS](https://css-tricks.com/css-only-carousel/) or use CSS framework like [bootstrap](https://getbootstrap.com/docs/5.2/components/carousel/).

here
![image](https://user-images.githubusercontent.com/63306833/174037702-bce634f6-9945-4cf8-8e9d-0405b4daef23.png)

and here

![image](https://user-images.githubusercontent.com/63306833/174037788-3b645ec4-d8e9-4740-a79a-626e2755565f.png)

- #### arrow looks strange, find better position

![image](https://user-images.githubusercontent.com/63306833/174038643-06bee5df-395c-470f-849c-a9ab0bf34883.png)

BudnikOleksii commented:
Thank you, guys, for the advice🧡 I will work on that)

BudnikOleksii commented:
> * #### U can create better hover effects, try to find something interesting on google
> 
> * #### Something wrong with the image background, u downloaded not right file type, this one is without transparent background
> 
> * #### Something wrong with the container
> 
> * #### Be more attentive to this menu, the cross is white, the text is dark, and what arrow doing here? And the gap between nav items looks streng
1) Updated hover effect for menu links.
2) The same situation with PNG, download image with filter from figma, looks better.
3) Fixed container size and margins.
4) Added script that arrow shows up only after 1200px scroll, is it ok idea?)
> * #### arrow looks strange, find better position
I tried different positions, but have no idea where it will be better😅 In the footer maybe it looks better, but I actually I want the arrow follows up user. 

For Ivan. Thank you, for the review, I think I fixed the bug with container in "FEATURES" section. Will work with animations.

Thank you so much, guys, I appreciate your advice!

I will work with hovers, sliders and think about how to get better position for the arrow😅

BudnikOleksii commented:
I added a slider to about-us(the first section with a slider on mobiles), slides position looks weird when you use it in devtools, but on two smartphones all is ok.
When you have time, please, check and advice is it ok?) I will keep working on it.

VETALEK reviewed (approved):
IMHO: i'd add some `border-radius` and make shadow more transparent 😉

<img width="546" alt="image" src="https://user-images.githubusercontent.com/55802630/174537331-d594dad0-cf8c-401c-abb9-5e47c244385d.png">

BudnikOleksii commented:
> IMHO: i'd add some `border-radius` and make shadow more transparent 😉
> 
Thank you, I have been playing a lot of time with different shadows and hovers and still trying them) It's fun, I will try to add border-radius and check it
